Data/Sys/GameSettings/GH2.ini

```diff
@@ -2,11 +2,13 @@
 
 [Core]
 # Values set here will override the main Dolphin settings.
+CPUThread = True
```

Comment:
I think there is no need to enforce dual core. It should be enabled by default by all users already, everyone else wants to get those slowdowns.